.github/workflows/amplify_api.yaml

```diff
@@ -7,30 +7,34 @@ on:
       - stable
   pull_request:
     paths:
-      - '.github/workflows/flutter_vm.yaml'
       - '.github/workflows/amplify_api.yaml'
+      - '.github/workflows/flutter_vm.yaml'
+      - 'packages/amplify/amplify_flutter/lib/**/*.dart'
+      - 'packages/amplify/amplify_flutter/pubspec.yaml'
+      - 'packages/amplify_core/lib/**/*.dart'
+      - 'packages/amplify_core/pubspec.yaml'
+      - 'packages/amplify_datastore_plugin_interface/lib/**/*.dart'
```

Comment:
I don't understand why this is here (realizing you did not introduce it here).

Reply:
amplify_api depends on amplify_flutter and amplify_flutter depends on amplify_datastore_plugin_interface. I don't think amplify_flutter needs to depend on amplify_datastore_plugin_interface though. I opened a PR to remove it https://github.com/aws-amplify/amplify-flutter/pull/3015